api/envoy/extensions/load_balancing_policies/subset/v3/subset.proto

```diff
@@ -18,7 +18,6 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: Subset Load Balancing Policy]
 // [#extension: envoy.load_balancing_policies.subset]
-// [#not-implemented-hide:]
```

Comment:
Should this stay hidden until we're ready to deprecate the originals in `envoy.config.cluster.v3.Cluster`?

Reply:
Then, how can the end users know now we can configure the new lb as extension in the `load_balancing_policy`?

And other LBs (ring hash/least request/etc) has eliminated the annotation. So, I think it would be better just keep two versions support for now.

And then, we can update example, docs, and related things, to guide users to use new API. And some PRs would be necessary to frozen old version API for a while. And finally, we can deprecate the old version API.

Considering the old version API is widely used and is very important for all users, I think this will be a long cycle to avoid bring much impact to users.